backend/src/routes/verse.rs

```diff
@@ -0,0 +1,25 @@
+use rocket;
+
+use crate::{guards::db_guard::DbGuard, services};
+
+#[rocket::get("/verse/<book>/<chapter>/<verse>")]
```

Comment:
Thoughts on using a query string here instead? Like verse?book=1&chapter=1&verse=1
```suggestion
#[rocket::get("/verse?<book>&<chapter>&<verse>")]
```
https://rocket.rs/v0.5-rc/guide/requests/#query-strings

Reply:
Good idea, it's more descriptive using queries. But the reason why I'm inclined to stick with path params is because they act as a directory structure. Since the Bible books are indexed based on a directory structure, it seems logical to have them as path params. Thoughts on my thoughts @j-koziel?

Reply:
I like your thoughts 🫶

Let's stick with this 👍